### DISCUSSION TITLE
How to change filesystem size on iOS iSH App ?

### DISCUSSION THREAD

**Discussion by eamadit** [ish-app/ish#1875] [Q&A]
Good Day,

I cannot copy files larger than 550Mb in the embedded Alpine Linux filesystem in the iOS iSH App. I suppose the filsystem is not large enough to do it. Is there a way to change its size ?

Best regards,

Camille

**Comment by tower-town** [comment]
I don't think there seems to be this specific application scenario. If you try to download a large file to the outside, you can mount the file.

Mounting external files can view this issue. https://github.com/ish-app/ish/issues/2139
